fields/civimembershiplevels.php

```diff
@@ -15,72 +15,70 @@
-	 * @since	1.6
-	 */
-	protected function getInput()
-	{
-		// Initialize variables.
-		$html = array();
-		$attr = '';
+  /**
+   * Method to get the field input markup.
+   *
+   * @return   string    The field input markup.
+   * @since    1.6
+   */
+  protected function getInput() {
+    // Initialize variables.
+    $html = array();
+    $attr = '';
 
-		// Initiate CiviCRM
-		require_once JPATH_ROOT.'/'.'administrator/components/com_civicrm/civicrm.settings.php';
-		require_once 'CRM/Core/Config.php';
-		$config =& CRM_Core_Config::singleton( );
+    // Initiate CiviCRM
+    require_once JPATH_ROOT . '/' . 'administrator/components/com_civicrm/civicrm.settings.php';
+    require_once 'CRM/Core/Config.php';
+    $config =& CRM_Core_Config::singleton();
 
-		// Initialize some field attributes.
-		$attr .= $this->element['class'] ? ' class="'.(string) $this->element['class'].'"' : '';
+    // Initialize some field attributes.
+    $attr .= $this->element['class'] ? ' class="' . (string) $this->element['class'] . '"' : '';
 
-		// To avoid user's confusion, readonly="true" should imply disabled="true".
-		if ( (string) $this->element['readonly'] == 'true' || (string) $this->element['disabled'] == 'true') {
-			$attr .= ' disabled="disabled"';
-		}
+    // To avoid user's confusion, readonly="true" should imply disabled="true".
+    if ((string) $this->element['readonly'] == 'true' || (string) $this->element['disabled'] == 'true') {
+      $attr .= ' disabled="disabled"';
+    }
 
-		$attr .= $this->element['size'] ? ' size="'.(int) $this->element['size'].'"' : '';
-		$attr .= $this->multiple ? ' multiple="multiple"' : '';
+    $attr .= $this->element['size'] ? ' size="' . (int) $this->element['size'] . '"' : '';
+    $attr .= $this->multiple ? ' multiple="multiple"' : '';
 
-		// Initialize JavaScript field attributes.
-		$attr .= $this->element['onchange'] ? ' onchange="'.(string) $this->element['onchange'].'"' : '';
+    // Initialize JavaScript field attributes.
+    $attr .= $this->element['onchange'] ? ' onchange="' . (string) $this->element['onchange'] . '"' : '';
 
-		// Get the field options.
-		$options = array();
-    $options[] = JHTML::_('select.option', '0', JText::_('- Select Membership Level -') );
-		$query = 'SELECT id, name FROM civicrm_membership_status WHERE is_active = 1 ORDER BY id';
-    $dao = CRM_Core_DAO::executeQuery( $query );
-    while ( $dao->fetch( ) ) {
-      $options[] = JHTML::_( 'select.option', $dao->id, $dao->name );
+    // Get the field options.
+    $options = array();
+    $options[] = JHTML::_('select.option', '0', JText::_('- Select Membership Level -'));
+    $query = 'SELECT id, name FROM civicrm_membership_status WHERE is_active = 1 ORDER BY id';
+    $dao = CRM_Core_DAO::executeQuery($query);
+    while ($dao->fetch()) {
+      $options[] = JHTML::_('select.option', $dao->id, $dao->name);
     }
-	
-		// Create a read-only list (no name) with a hidden input to store the value.
```

Comment:
why are you removing this? I haven't fully traced the code. can you explain what you found?